client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -23,20 +26,23 @@ export default () => {
 
 	const homepageTemplates = templates.filter( template => template.category === 'home' );
 
-	const [ previewedTemplate, setPreviewedTemplate ] = useState< string | null >( null );
+	const [ selectedDesign, setSelectedDesign ] = useState< string | undefined >();
 	return (
-		<div
-			className="page-template-modal__list" // eslint-disable-line wpcalypso/jsx-classname-namespace
-		>
-			<TemplateSelectorControl
-				label={ NO__( 'Layout', 'full-site-editing' ) }
-				templates={ homepageTemplates }
-				blocksByTemplates={ {} /* Unneeded, since we're setting `useDynamicPreview` to `false` */ }
-				onTemplateSelect={ setPreviewedTemplate }
-				useDynamicPreview={ false }
-				siteInformation={ undefined }
-				selectedTemplate={ previewedTemplate }
-			/>
+		<div className="design-selector">
+			{ homepageTemplates.map( t => (
+				<Card
+					className={ classnames( 'design-selector__design-option', {
+						'is-selected': t.slug === selectedDesign,
+					} ) }
+					key={ t.slug }
+					isElevated
+					onClick={ () => setSelectedDesign( t.slug ) }
+				>
+					<CardMedia>
+						<img alt={ t.title } src={ t.preview } />
```

Comment:
I think we might add a `srcset` in a follow-up. Maybe I'll turn [this stuff](https://github.com/Automattic/wp-calypso/pull/38403/files#diff-b43d337b3bf37258ebc7f894a3e9b6a5R30) into a util function that returns everything needed for a `srcset`.

Reply:
There's also some prior art for constructing photon-style srcsets in Jetpack Tiled Gallery block:

https://github.com/Automattic/jetpack/blob/ed104a24c0a73f6015abcff1b01d0d3b693b52f3/extensions/blocks/tiled-gallery/utils/index.js#L72-L111

Reply:
Thanks! I think we're dealing with a tighter set of constraints here (already-photonized WP.com images), so I was hoping to get away with a [rather simple implementation](https://github.com/Automattic/wp-calypso/pull/38465/files#diff-b43d337b3bf37258ebc7f894a3e9b6a5R22)...